src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py

```diff
@@ -38,24 +32,23 @@ class ElasticJobParametersDB(ElasticDB):
     def __init__(self, parentLogger=None):
         """Standard Constructor"""
 
-        try:
-            indexPrefix = CSGlobals.getSetup().lower()
+        self.fullname = "WorkloadManagement/ElasticJobParametersDB"
+        self.index_name = self.getCSOption("index_name", "job_parameters")
 
+        try:
             # Connecting to the ES cluster
-            super().__init__("WorkloadManagement/ElasticJobParametersDB", indexPrefix, parentLogger=parentLogger)
+            super().__init__(self.fullname, self.index_name, parentLogger=parentLogger)
         except Exception as ex:
             self.log.error("Can't connect to ElasticJobParametersDB", repr(ex))
```

Comment:
If we get here, then `self.log` might not exist because we get the log from grand parent class?!?

Reply:
Well spotted. Removed.